panacea/tx.go

```diff
@@ -20,6 +21,17 @@ func NewTxBuilder(client QueryClient) *TxBuilder {
 	}
 }
 
+// GenerateTxBytes generates transaction byte array.
+func (tb TxBuilder) GenerateTxBytes(privKey cryptotypes.PrivKey, conf *config.Config, msg ...sdk.Msg) ([]byte, error) {
+	defaultFeeAmount, _ := sdk.ParseCoinsNormalized(conf.Panacea.DefaultFeeAmount)
```

Comment:
I couldn't remember why I skipped to handle error here.
Why don't you handle error here as well?

Reply:
I don't know why skip to handle error either. @gyuguen Is there any reason?

Reply:
I might write the code.. 😂. I don't remember..

Reply:
I simply copied this code.
But it seems to wrong code. 
We need to handling error. 😂

Reply:
So, I just fixed it . (https://github.com/medibloc/panacea-doracle/pull/91/commits/1467b54b33191b8e50ac6d880482a97874510337)

Reply:
Okay, I fixed it, too. Thanks!

Reply:
Thank you guys!